lib/runtime/console.coffee

```diff
@@ -72,11 +72,15 @@ module.exports =
   info: (data) -> @c.info data
 
   toggle: ->
-    d = atom.workspace.paneContainerForItem(@c)
-    if !d
+    dock = atom.workspace.paneContainerForItem(@c)
+    if dock and dock.toggle
+      if dock.isVisible()
+        dock.hide()
+      else
+        dock.show()
+        @c.open()
```

Comment:
Not really, because we need to call `@c.open()` only when opening the dock. If we don't then the console won't be focused.